Description: List only supported names for ksh
 The Korn Shell will never be the default shell
 on Debian, so we strip out references to that.
 The pfksh variant is solaris specific so we strip
 out references to that.
Author: Oliver Kiddle <okiddle@yahoo.co.uk>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=463022
Forwarded: no
Last-Update: 2015-11-01
--- a/src/cmd/ksh93/sh.1
+++ b/src/cmd/ksh93/sh.1
@@ -19,15 +19,7 @@
 .TH KSH93 1
 .\}
 .SH NAME
-.if \nZ=0 \{\
-sh, rsh, pfsh \- shell, the standard/restricted command and programming language
-.\}
-.if \nZ=1 \{\
-ksh, rksh, pfksh \- KornShell, a standard/restricted command and programming language
-.\}
-.if \nZ=2 \{\
-ksh93, rksh93, pfksh93 \- KornShell, a standard/restricted command and programming language
-.\}
+ksh, rksh, ksh93, rksh93 \- KornShell, a standard/restricted command and programming language
 .SH SYNOPSIS
 .if \nZ=0 \{\
 .B sh
@@ -67,35 +59,21 @@
 .B \-
 ] [ arg .\|.\|. ]
 .SH DESCRIPTION
-.if \nZ=0 .I Sh\^
 .if \nZ=1 .I Ksh\^
 .if \nZ=2 .I Ksh93\^
 is a command and programming language
 that executes commands read from a terminal
 or a file.
-.if \nZ=0 .I Rsh\^
 .if \nZ=1 .I Rksh\^
 .if \nZ=2 .I Rksh93\^
 is a restricted version of the
 .if \nZ=0 standard
 command interpreter
-.if \nZ=0 .IR sh ;
 .if \nZ=1 .IR ksh ;
 .if \nZ=2 .IR ksh93 ;
-it is used to set up login names and execution environments whose
-capabilities are more controlled than those of the standard shell.
-.if \nZ=0 .I Rpfsh\^
-.if \nZ=1 .I Rpfksh\^
-.if \nZ=2 .I Rpfksh93\^
-is a profile shell version of the
-.if \nZ=0 standard
 command interpreter
-.if \nZ=0 .IR sh ;
 .if \nZ=1 .IR ksh ;
 .if \nZ=2 .IR ksh93 ;
-it is used to to execute commands with the attributes specified by
-the user's profiles (see
-.IR pfexec (1)).
 See
 .I Invocation\^
 below
@@ -2168,12 +2146,6 @@
 or
 .BR krsh ,
 then the shell becomes restricted.
-If it is
-.BR pfsh
-or
-.BR pfksh ,
-then the shell becomes a profile shell (see
-.IR pfexec (1)).
 .TP
 .SM
 .B TIMEFORMAT
@@ -7945,15 +7917,6 @@
 This set of strings will be subject to language translation
 when the locale is not C or POSIX.
 No commands will be executed.
-.PD
-.TP
-.B \-P
-If
-.B \-P
-or
-.B \-o profile
-is present, the shell is a profile shell (see
-.IR pfexec (1)).
 .TP
 .BI \-R " filename\^"
 The
@@ -8122,7 +8085,6 @@
 gmacs(1),
 grep(1),
 newgrp(1),
-pfexec(1),
 stty(1),
 test(1),
 umask(1),
